tests/test_content.py

```diff
@@ -240,6 +240,21 @@ def test_content_subreddit_from_name(reddit, terminal):
     assert content.name == '/r/python'
     assert content.order is None
 
+    name = '/domain/python.org'
```

Comment:
Since this test requires HTTP requests to run, you will need to generate a new cassette in order to get it to pass. Check out the section in the README faq about running tests and let me know if you have any questions.